cement/ext/ext_smtp.py

```diff
@@ -62,17 +74,35 @@ def _get_params(self, **kw: Any) -> Dict[str, Any]:
         params = dict()
 
         # some keyword args override configuration defaults
-        for item in ['to', 'from_addr', 'cc', 'bcc', 'subject',
-                     'subject_prefix', 'files']:
+        for item in [
+            'to', 'from_addr', 'cc', 'bcc', 'subject', 'subject_prefix', 'files',
+            'charset', 'header_encoding', 'body_encoding',
+            'date_enforce', 'msgid_enforce', 'msgid_str',
+        ]:
             config_item = self.app.config.get(self._meta.config_section, item)
             params[item] = kw.get(item, config_item)
 
         # others don't
-        other_params = ['ssl', 'tls', 'host', 'port', 'auth', 'username',
-                        'password', 'timeout']
-        for item in other_params:
-            params[item] = self.app.config.get(self._meta.config_section,
-                                               item)
+        for item in [
```

Comment:
I changed all loops looking the some coding as you did in the first `for item in []`